src/constants.js

```diff
@@ -27,6 +27,8 @@ export const TestDataLocations = {
 
 export const ModelNameMaxLength = 150;
 
+export const UNIQUE_OPTIONS_MAX = 50;
```

Comment:
50 is kind of arbitrary - it's smaller than it would need to be to alleviate the technical concern and still large enough that it doesn't guarantee high model accuracy.  I think it probably warrants a conversation about how restrictive we'd like to be from the curricular/product perspective?

Reply:
@moneppo how does this cut-off number feel as related to the new datasets you're finding?

Reply:
Clarifying that this would only affect *categorical* data, yes? For example, numerical would definitely have more than 50 unique values, but that's expected since we want to train on the range.

If this is only affecting categorical data, then 50 options max feels fine to me curricularly. I'd defer to Mike on what he's observed in the datasets he's been collecting if that feels off from his experiences.

Reply:
D'oh! I just saw the screenshot that says "categorical columns...". Please disregard my question above =P